extensions/miniflux/src/open-miniflux.tsx

```diff
@@ -1,6 +1,11 @@
-import { getPreferenceValues, open, popToRoot } from "@raycast/api";
+import { getPreferenceValues, open, PreferenceValues } from "@raycast/api";
 
 export default async function OpenMiniflux() {
-  const { baseUrl }: Preferences = getPreferenceValues();
-  baseUrl ? open(baseUrl) : popToRoot();
+  const { baseUrl }: PreferenceValues = getPreferenceValues();
+
+  if (baseUrl) {
+    open(baseUrl);
+  }
+
+  return null;
```

Comment:
The `return null` is not needed anymore since you changed this command to `no-view`.

See https://developers.raycast.com/information/lifecycle#launch.

Also, the `async` is unnecessary since you didn't use the `await`. Or you can add `await` to the `open()` function.